styles.css

```diff
@@ -678,4 +678,549 @@ main hr {
+  }
+
+  .welcome-section h2 {
+    width: 100%;
+    font-weight: 500;
+    font-size: 2rem;
+    line-height: 44px;
+    margin-top: 24px;
+  }
+
+  .welcome-section p {
+    font-family: 'Inter', sans-serif;
+    width: 100%;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    margin-top: 24px;
+  }
+
+  .social-icons {
+    flex-direction: column;
+    width: 100%;
+    flex-basis: 5%;
+    margin-left: 36px;
+    order: -1;
+  }
+
+  .social-icons i {
+    margin-top: 20px;
+  }
+
+  .illustration-header-mobile {
+    display: none;
+  }
+
+  .illustration-header-desktop {
+    display: flex;
+    justify-content: right;
+    flex-basis: 50%;
+    height: 100%;
+    transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
+    position: relative;
+    left: 5%;
+  }
+
+  #vectors-illustration-desktop {
+    max-height: 100%;
+    max-width: 100%;
+    width: 44.7vw;
+    height: 43.9vw;
+    transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
+  }
+
+  /* Here ends the welcome page */
+
+  /* Here starts daily selection section */
+
+  .works {
+    padding-top: 135px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    overflow: hover;
+  }
+
+  .works-title {
+    justify-content: center;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .works-title-container {
+    flex-direction: row;
+    align-items: center;
+    width: 78.68vw;
+    justify-content: center;
+    margin-top: 135px;
+  }
+
+  .works-divider {
+    width: 55%;
+    margin-left: 1.6vw;
+    margin-bottom: 12px;
+    color: #212121;
+  }
+
+  .daily-selection {
+    height: 31vw;
+    margin-top: 100px;
+    width: 80.2vw;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    max-width: 1445px;
+    max-height: 560px;
+    min-width: 1040px;
+  }
+
+  .img-placeholder-mobile {
+    display: none;
+  }
+
+  .img-placeholder-desktop {
+    flex-basis: 56%;
+    width: 45vw;
+    height: 31vw;
+    display: flex;
+    align-items: center;
+    margin-right: 1.6vw;
+    max-width: 1445px;
+    max-height: 560px;
+  }
+
+  .img-daily-desktop {
+    width: 100%;
+    height: 100%;
+  }
+
+  .daily-selection article {
+    flex-basis: 44%;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+    align-items: flex-start;
+    gap: 1.92%;
+  }
+
+  .daily-selection article h3 {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .daily-selection p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .tags {
+    width: 1.57vw;
+    height: 2.7vw;
+    gap: 20px;
+  }
+
+  #desktop-ul li {
+    padding-right: 0.83vw;
+    padding-left: 0.83vw;
+    padding-bottom: 0.69vw;
+    padding-top: 0.69vw;
+  }
+
+  .see-more-container {
+    width: 22.7vw;
+    margin-bottom: 24px;
+  }
+
+  .see-more-btn {
+    width: 8.6vw;
+    height: 3.3vw;
```

Comment:
- [ ] Buttons when pressed don't have the same color as on the design, kindly update it
![image](https://user-images.githubusercontent.com/62705947/186944237-b3e799ab-1c34-4bed-b6b4-1c0e6ae11375.png)